api/tests/views_tests.py

```diff
@@ -598,9 +606,10 @@ def _setup_client(token):
     return client
 
 
-@pytest.mark.django_db
 @responses.activate
-def test_duplicate_email_logs_details_for_debugging(caplog: pytest.LogCaptureFixture):
+def test_duplicate_email_logs_details_for_debugging(
+    caplog: pytest.LogCaptureFixture, fxa_social_app: SocialApp
```

Comment:
question (non-blocking): This test invokes the `fxa_social_app` fixture by adding it to the parameters of the test, but the test doesn't reference the `fxa_social_app` value. Why use a parameter here instead of using `@pytest.mark.usefixtures` like above?

Reply:
Interesting idea, I've only used it on classes. I'll try it.

Reply:
It worked, this may be the new pattern for "apply but don't use this fixture"